conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
> especially where we will have an undefined behavior when es6 is false and es7 is true. What's expected then?

I got it, I felt better that we don't add `es7` environment for now.
I think we need to redesign about built-in global variables in future.
E.g. `parserOptions.ecmaVersion` decides built-in global variables.